requirements/requirements-all.txt

```diff
@@ -1,8 +1,8 @@
 sqlalchemy>=2.0.0
 logmuse
-peppy>=0.35.6
+peppy==0.35.6
 ubiquerg>=0.6.2
 coloredlogs>=15.0.1
 pytest-mock
-pydantic
-psycopg2
+pydantic<2.0
+psycopg2-binary
```

Comment:
back to binary?

Reply:
yes,
I can't install psycopg2, @ayobi has the same issue

Reply:
Sounds good